apps/backend/src/mcp/tools/device_management.py

```
@@ -17,7 +17,7 @@
 from apps.backend.src.core.database import get_async_session
 from apps.backend.src.models.device import Device
 from apps.backend.src.core.exceptions import (
-    DeviceNotFoundError, DatabaseOperationError, SSHConnectionError
+    DeviceNotFoundError, DatabaseOperationError, SSHConnectionError, ValidationError
```

Ruff (0.12.2)

`apps.backend.src.core.exceptions.SSHConnectionError` imported but unused

Remove unused import: `apps.backend.src.core.exceptions.SSHConnectionError`

(F401)